canopeum_frontend/.prettierignore

```diff
@@ -0,0 +1 @@
+**
```

Comment:
Not using vscode. This is normally the recommended solution for prettier users. If you want I can add it to the gitignore so that it isn't commited

Reply:
If we have to add more acknowledgements of prettier despite not using it, I'd rather have the file that disables it instead of adding it to .gitignore and still having issues from other contributors running it automatically.

But let's do that in another PR.

I'm also curious which editor you're using and if it supports pre-project/workspace configurations.